search_username/catalog/views.py

```diff
@@ -0,0 +1,46 @@
+from django.shortcuts import render, redirect
+from InstagramAPI import InstagramAPI
+from search_username.settings import password, user_name
+
+
+class GetInstagramInfo:
+    def __init__(self, username, password):
+        self.api = InstagramAPI(username=username, password=password)
+        self.api.login()
+
+    def get_user_id(self, name):
+        """Returns id user."""
+        self.api.searchUsername(name)
+        return self.api.LastJson["user"]["pk"]
+
+    def get_followers(self, user_id=None):
+        """Returns list of user's followers."""
+        user_id = user_id or self.api.username_id
```

Comment:
Well, here I'm not sure if we really need to return list of followers from the logged in user.

Reply:
really ,deleted